### PR TITLE
Deployment tutorial with scp and surge.sh instructions

### PR DESCRIPTION
**Pull request by boztek**
Addresses issue #1022.

### CODE REVIEW

File: source/localizable/tutorial/deploying.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+To deploy an Ember application simply transfer the output from `ember build` to a web server. This can be done with standard Unix file transfer tools such as `rsync` or `scp`. There are also services that will let you deploy easily.
```
Comment (locks) [inline source/localizable/tutorial/deploying.md]: Hi @boztek, can you split up the sentences as specified in https://github.com/emberjs/guides/blob/master/CONTRIBUTING.md#style-guide ? It makes it easier to review and diff. Appreciated :)

**Comment by acorncom** [comment]
Also, we tend to squash commute on ththe se types of changes. Once that's done, we'll merge this! :smile:

**Comment by boztek** [comment]
Squash commits into one you mean @acorncom ?

**Comment by boztek** [comment]
Done. Thanks @locks and @acorncom.

**Comment by locks** [comment]
@homu r+

**Pull request merged**